### ISSUE TITLE
[BUG]: different layouts for every category (ie. application, marketing)

### ISSUE THREAD

**Issue by zoltanszogyenyi** [shinokada/flowbite-svelte-blocks#15]
### Description of the bug

Hey @shinokada,

I'm doing a PR on this to improve the UI/UX of the layouts - I've already modified the homepage navigation and footer, but why are there multiple navbars for every category? We should only have one `layout.svelte` with the same navbar and footer.

I have to make the changes in 3-4 places now and maybe even more in the future - the navbar classes have been properly added in the hompage file for now with the corrent container widths (ie. `max-w-8xl`).

Also, I try to update the layout for the blocks but again, why don't we have one single component for the whole block? We can just pass props like the title, description, GitHub edit link, source code etc.

Right now I want to update the layout of the block to something like this:

<img width="1443" alt="Screenshot 2023-03-25 at 14 46 36" src="https://user-images.githubusercontent.com/8052108/227718116-4d400513-eb39-4cb6-b040-f3bfd543909a.png">

Correct me if I'm mistaken but right now to do this I have to apply it 132 times which is crazy.

I think that we need to optimize some of these components because maintaining the library will become very hard.

Otherwise, great job on the repo!

Cheers,
Zoltan

### Steps To Reproduce

1. Install the project
2. Make changes
3. Having to make changes in hundreds of places because the layout components are not re-used

**Comment by shinokada** [comment]
Yes, you are right. We can use +layout.svelte for all category layouts.
